frontend/views/pages/PayGroup.vue

```diff
@@ -3,14 +3,21 @@ page(
   pageTestName='payGroupPage'
```

Comment:
@pieer you are doing a great job here handling all the details and edge cages around payments, not an easy task!

Obviously, this file is getting ginormous and it's still missing some UI and connecting to the store. The bigger it gets, the harder it will be to modify/catch a bug in the future.

Because of that, I think we should extract some parts of the page into smaller components, maybe the sidebar, the search, and the footer, and focus only on the "tab/table view".

Another concern I have, but already discussed during the hackathon when implementing "Income Details" page: I think the amount of if statements and ternary around is cluttering the component markup way too much. I gave a suggestion on a segment of the component, let me know on that component what you think.

Reply:
I agree, that was only a first version

Reply:
Awesome, was just to make sure we are on the same page 😊